.github/ISSUE_TEMPLATE/Distribution_Request.md

```diff
@@ -17,16 +17,19 @@ assignees: ''
     - [ ] Tar.gz
     - Snap - Please file issue in https://github.com/powershell/powershell-snap.  This issues type is unrelated to snap packages with a distribution neutral.
 - Processor Architecture (One per request):
-- [ ] **Required** - An issues has been filed to create a Docker image in https://github.com/powershell/powershell-docker
 - The following is a requirement for supporting a distribution **without exception.**
-    - [ ] The version and architecture of the Distribution is [supported by .NET Core](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md#linux).
+    - [ ] The version and architecture of the Distribution is [supported by .NET Core](https://github.com/dotnet/core/blob/master/release-notes/5.0/5.0-supported-os.md#linux).
 - The following are requirements for supporting a distribution.
   Please write a justification for any exception where these criteria are not met and
   the PowerShell committee will review the request.
     - [ ] The version of the Distribution is supported for at least one year.
     - [ ] The version of the Distribution is not an [interim release](https://ubuntu.com/about/release-cycle) or equivalent.
 
-## Progress - For PowerShell Team **ONLY**
+## Progress
+
+- [ ] An issues has been filed to create a Docker image in https://github.com/powershell/powershell-docker
```

Comment:
Is it a mandatory field to be filled by the issue owner?

Reply:
I moved it to progress to say it is not mandatory for the owner, just suggested to prevent delay.